wiki/Beatmap/Converts/en.md

```diff
@@ -0,0 +1,48 @@
+---
+tags:
+  - convert beatmaps
+  - converted beatmaps
+---
+
+# Converts
+
+![The \"Show converted\" toggle in osu!(lazer)](img/show-converted-button.png "The \"Show converted\" toggle in osu!(lazer)")
+
+In osu! gameplay, **converts** (or *converted beatmaps*) are [beatmaps](/wiki/Beatmap) of a specific [game mode](/wiki/Game_mode) that have been "converted" by the osu! client in-game in order to be playable in another game mode they were not originally created for. All converts are automatically generated based on certain parameters present in the base beatmap.
+
+As it stands, osu! supports conversions of beatmaps from the [osu! game mode](/wiki/Game_mode/osu!) into [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania).
+
+## Conversions
+
+### osu! to osu!taiko conversion
+
+*See also: [osu!taiko mapping/osu! conversion notes](/wiki/Game_mode/osu!taiko#osu!-conversion-notes)*
+
+Converts between osu! and osu!taiko are generated according to the rhythm, hitsound, and slider velocity changes in the beatmap. Depending on how the [hit objects](/wiki/Gameplay/Hit_object) are being hitsounded, the conversion between the two game modes generally goes as follows:
```

Comment:
"converts between X and Y" gives an impression that it could be both way -- why not write "from X to Y" instead?

Reply:
also I would say hitsounds play the most important role here, so one sentence instead of two:
```suggestion
osu!taiko converts are generated according to the placement of hitsounds, rhythm, and slider velocity changes in a beatmap:
```

Reply:
changed the first sentence but i'll be keeping the "generally goes as follows" part i think, as (to my knowledge) there are some edge cases where the conversion is happening differently